.unacceptablelanguageignore

```diff
@@ -0,0 +1,3 @@
+IntegrationTests/run-tests.sh
```

Comment:
Can you use line based ignores here instead please by adding `# ignore-unacceptable-language` to the lines.

Reply:
Fixed.